README.md

```diff
@@ -16,6 +16,7 @@ Set these environment variables if you need to change their defaults
 | ----------------------------- | ----------------------------------------------------------------- | -------------- |
 | TEMPORAL_GRPC_ENDPOINT        | String representing server gRPC endpoint                          | 127.0.0.1:7233 |
 | TEMPORAL_WEB_PORT             | HTTP port to serve on                                             | 8088           |
+| TEMPORAL_CA_FILE              | TLS certificate for secure connection to temporal server          |                |
```

Comment:
CA cert is only to validate frontends cert in case it is issues by a non-public CA. Also need to set server name to be validated against the frontend's cert (and enable host name verification).
For mTLS also need to provide a cert and a private key for the web server.

Reply:
updated the pr. Also checking the part of verification with the server name